### ISSUE TITLE
Problem of dataset

### ISSUE THREAD

**Issue by hbfast** [Fanghuachen/AEDNet#1]
In the dataset, I found the range of polarity is from zero to one. Do these values mean magnitude of illumination variations?

**Comment by Fanghuachen** [comment]
The polarity is grayscale value. The value is from 0 to 255 and then normalize to 0-1.

**Comment by wwj-53** [comment]
There are 2 warning in dataprocess.py ：
Cannot find reference 'shape_patch_count' in 'Sized | None'
Cannot find reference 'shape_names' in 'Sized | None'
Will this really not affect program functionality?

**Comment by wwj-53** [comment]
May I ask if the hyperparameters in your open source code are the final hyperparameters? Because I have been unable to achieve the results stated in the paper. I would like to know which ones your training set has been selected from Simulated_data in DVSClean, could you please let me know? This has been very helpful to me. Thank you very much for your contributions.

**Comment by Fanghuachen** [comment]
The uploaded hyperparameters are the final hyperparameters. I want to know in which dataset you are unable to achieve the results stated in the paper. For DVSNOISE20, you should train the model individually on DVSNOISE20, since the spatial resolution is different from that of DVSCLEAN. For the DVSCLEAN, I set scenes with prefix MAH00447, MAH00451, MAH00456, MAH00459, MAH00462, MAH00468, MAH00489, MAH00490, MAH00508 and MAH00512 as verification set and other scenes as training set. I hope this can help you. Other people who emailed me have achieved the stated results, therefore I think the uploaded hyperparameters are the correct version. Please recheck your operation and try it again. If you have other questions, feel free to contact us. I'm willing to help you resolve your puzzle.
















At 2024-03-01 10:52:03, "wwj-53" ***@***.***> wrote:

May I ask if the hyperparameters in your open source code are the final hyperparameters? Because I have been unable to achieve the results stated in the paper. I would like to know which ones your training set has been selected from Simulated_data in DVSClean, could you please let me know? This has been very helpful to me. Thank you very much for your contributions.

**Comment by JiangJie37** [comment]
> The polarity is grayscale value. The value is from 0 to 255 and then normalize to 0-1.

Hello, I don't quite understand, isn't the polarity only 0/1? Why are gray values used here? Is the network disabled if the event-only camera is used?
